README.md

```diff
@@ -51,5 +51,12 @@ Please find a screenshot of the expected results below:
 
 ## Extra Resources
 
-- [CSS Reference from MDN](https://developer.mozilla.org/en-US/docs/Web/CSS)
+Below is a gif of how the content looked on a 1400x800 MacBook type resolution running on Chrome Browser.
+- ![Progress Gif](https://i.gyazo.com/4f58a262d20f037295f7f0c0ef93dfc1.gif)
+
+Here is a small video of the site running on mobile resolutions.
```

Comment:
Don't worry about making your webpage responsive.  We haven't gone over how to handle that yet.